ARK Server Manager/Lib/Model/ServerProfile.cs

```diff
@@ -1860,6 +1860,8 @@ public string GetServerArgs()
                 serverArgs.Append($"?GameModIds={this.ServerModIds}");
             }
 
+            if (!String.IsNullOrWhiteSpace(this.AdditionalArgs) && !this.AdditionalArgs.StartsWith(" "))
```

Comment:
At the moment if the user has added additional argument, then yes there is space added beforehand, otherwise it did not work.
The problem that was raised is that a space was automatically added by ASM beforehand and the user had to.